distributions/openhab/src/main/resources/runtime/etc/jetty.xml

```diff
@@ -23,7 +23,7 @@
 					<Call name="addRule">
 						<Arg>
 							<New class="org.eclipse.jetty.rewrite.handler.ForwardedSchemeHeaderRule">
-								<Set name="header">X-Forwarded-Scheme</Set>
+								<Set name="header">X-Forwarded-Proto</Set> <!-- RFC7239 states -Proto not -Scheme as header, Jetty seems to have wrong defaults -->
```

Comment:
I don't think these are Jetty defaults as such, probably no need for the comment here.

Reply:
They are in deed Jetty defaults, see [Jetty API documentation](http://archive.eclipse.org/jetty/8.1.0.RC0/apidocs/org/eclipse/jetty/rewrite/handler/ForwardedSchemeHeaderRule.html) (and I can't find that class name anywhere else in OpenHAB sources). So I would leave the comment, okay?

Reply:
Did you read RFC7239? It actually specifies a [Forwarded](https://tools.ietf.org/html/rfc7239#section-4) header to  replace and standardize headers like `X-Forwarded-Scheme` and `X-Forwarded-Proto`. Jetty doesn't define a default and allows you to use the `RewriteHandler` to use whatever header you want for this. 

So the comment doesn't make sense and for better compatibility and not breaking peoples existing setups you may want to define additional rules for this. People may currently have setup their NGINX/Apache to use this `X-Forwarded-Scheme` header instead of a `X-Forwarded-Proto` header.